test/suite/test_compat03.py

```diff
@@ -46,6 +46,7 @@ class test_compat03(wttest.WiredTigerTestCase, suite_subprocess):
     # Log version 1 does not have the prevlsn record.
     # Log version 2 introduced that record.
     # Log version 3 continues to have that record.
+    # Log version 4 continues to have that record.
```

Comment:
I know this went into the other PR. Maybe change both to read something like:
Log versions 3 and later continue to have that record.

So that we don't need to call that out every version. It is unlikely to go away.

Reply:
Done.